apps/eo/app-eo/project.json

```diff
@@ -56,22 +56,22 @@
             {
               "type": "allScript",
               "maximumWarning": "500kb",
-              "maximumError": "1mb"
```

Comment:
This is because of recent changes to the project, where new packages have added  to the overall size of the bundle :) It now excees the 1mb that was alotted, and is at 1.01. So in order for the CI to allow me to build, i have to increase the maximum allowed size.

Reply:
...or stop increasing the bundle size. Increasing the size limit is the road to Hell for our users. What's the cause? Is something not lazy loaded?

Reply:
I observed this after i pushed an update with ng2-charts, chart.js, and chartjs-plugin-datalabels :) I had just merged in an update from main that included "inputmask" package as well, and that caused my build to fail on this.

I am unsure how to procede from here. Any advice @LayZeeDK  or @Klemmensen ? I don't know how the app should behave after adding 4 new packages in total.

Reply:
We must analyze the output bundles using source-map-analyzer or [@ngx-builders/analyze](https://github.com/ngx-builders/source-map-analyzer).

We can also evaluate which budgets are going to grow with the app or how we can prevent this: [Configuring size budgets](https://angular.io/guide/build#configuring-size-budgets).